Vite note about vite compiling on the fly (#2083)
Scalingo/documentation: src/_posts/languages/nodejs/2000-01-01-start.md

  },
```

> As a note, `vite --port $PORT` may compile components, depending of your configuraiton. If it does, it will leads to a very long first http request that may even crash your container. In this case, in the `package.json`, replace the `start` rule content to use `node server.js` (see section `Framework Requiring to Serve Static Files`) or `vite preview --port $PORT`.

Reviewer: ```suggestion
As a note, `vite --port $PORT` may compile components, depending on your configuration. If it does, it leads to a very long first HTTP request that may even crash your container. In this case, in the `package.json`, replace the `start` rule content to use `node server.js` (see section `Framework Requiring to Serve Static Files`) or `vite preview --port $PORT`.
```
